manifests/init.pp

```diff
@@ -87,4 +124,9 @@
   contain 'opendkim::install'
```

Comment:
Remove contains

Reply:
Able to remove contain statement and now class included by absolute name. Now only operators ( -> and ~>) kept in the code